editor/editor_scale.cpp

```diff
@@ -32,12 +32,12 @@
 
 #include "core/os/os.h"
```

Comment:
While playing with it I saw this is unused, so can be removed.

(Still really missing some checks to remove-what-you-dont-use instead of the overkill include-what-you-use that just adds stuff but doesn't remove.)